.github/workflows/build_linux.yml

```diff
@@ -0,0 +1,52 @@
+name: Ubuntu Linux (x86_64)
+
+on:
+  push:
+    branches:
+      - "*"
+    tags:
+      - "*"
+
+  pull_request:
+    branches:
+      - "*"
+
+jobs:
+  build:
+    name: Build & Test Rust Crate
+    runs-on: ubuntu-latest
+
+    env:
+      RUST_BACKTRACE: full
+
+    steps:
+      - uses: actions/checkout@v3
+      
+      - name: Install Packages
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y libgl1-mesa-dev mesa-common-dev libsdl2-dev libglm-dev llvm-dev
```

Comment:
i'm sure we don't need llvm-dev right now

Reply:
Sounds good to me! There will be little things that will need to be correct, tuned. Thank you guys for getting into rust lol